client/src/components/existing-request/existing-request-display.vue

```diff
@@ -486,7 +490,11 @@ export default class ExistingRequestDisplay extends Mixins(NrAffiliationMixin, C
           await this.affiliateOrLogin()
           break
         default:
-          await newReqModule.patchNameRequestsByAction(action)
+          if (await newReqModule.patchNameRequestsByAction(action)) {
+            newReqModule.mutateDisplayedComponent('Success')
+            await sleep(1000)
+            newReqModule.mutateDisplayedComponent('ExistingRequestDisplay')
+          }
```

Comment:
Again, afte a successful patch, display the success component for 1 second then re-display THIS page.

Reply:
Is this to prevent seeing a state transition after the component is rendered? 
I've noticed a few times after returning from payment for example.

Reply:
It is to display the success component for 1000 ms and then go back to the ERD page.

What you've seen after returning from payment (which I didn't change) is the reload of the app, and then the retrieval and display of the NR and receipt. I'm sure that can be improved later :)